### PR TITLE
Crowbar rename

### PR DESCRIPTION
**Pull request by galthaus**

### CODE REVIEW

File: chef/cookbooks/kernel-panic/metadata.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name             'kernel-panic'
-maintainer       "Dell Crowbar Team"
```
Comment (zehicle) [inline chef/cookbooks/kernel-panic/metadata.rb]: We should leave Dell Crowbar and not change it to Dell Rebar.

File: chef/cookbooks/logging/metadata.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 name             'logging'
-maintainer       "Dell Crowbar Team"
+maintainer       "Dell Rebar Team"
```
Comment (zehicle) [inline chef/cookbooks/logging/metadata.rb]: should stay Dell Crowbar

File: chef/cookbooks/network/metadata.rb [hunk 3]
```diff
@@ -1,9 +1,9 @@
 name             'network'
-maintainer       "Dell Crowbar Team"
+maintainer       "Dell Rebar Team"
```
Comment (zehicle) [inline chef/cookbooks/network/metadata.rb]: should stay Dell Crowbar -> or just change the maintainer to be RackN Inc

File: chef/cookbooks/network/metadata.rb [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 maintainer_email "openstack@dell.com"
```
Comment (zehicle) [inline chef/cookbooks/network/metadata.rb]: this email is dead.  maybe change to digitalrebar@googlegroups.com or digitalrebar@rackn.com

File: chef/cookbooks/kernel-panic/metadata.rb (hunk 1 above)
Comment (galthaus) [inline chef/cookbooks/kernel-panic/metadata.rb]: Seems like we should remove Dell.  Dell Crowbar team isn't doing anything with this. :-)

Comment (ravolt) [inline chef/cookbooks/kernel-panic/metadata.rb]: +1

On 09/10/2015 10:36 AM, Greg Althaus wrote:
>
> In chef/cookbooks/kernel-panic/metadata.rb
> <https://github.com/digitalrebar/core/pull/803#discussion_r39174697>:
>
> > @@ -1,5 +1,5 @@
> >  name             'kernel-panic'
> > -maintainer       "Dell Crowbar Team"
>
> Seems like we should remove Dell. Dell Crowbar team isn't doing
> anything with this. :-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/digitalrebar/core/pull/803/files#r39174697>.
>

-- 

Rob
____________________________
Rob Hirschfeld, 512-773-7522

I am in CENTRAL (-6) time
http://robhirschfeld.com
twitter: @zehicle, github: zehicle & ravolt

**Pull request merged**